src/database/sequelize.ts

```diff
@@ -0,0 +1,17 @@
+import {databaseName, password, userName} from '../config/config';
+import {Sequelize} from 'sequelize';
+import userModel from '../database/user.model';
+ 
+const sequelize = new Sequelize(databaseName, userName, password, {
+  dialect: 'postgres',
+  host: 'localhost'

```

Comment:
You should use host variable from config

Reply:
This config object has interestig type for dialect
(property) Options.dialect?: Dialect
I was convenient to use it here

Reply:
Config variables are used now